Content.Server/Physics/Controllers/SingularityController.cs

```diff
@@ -1,106 +1,56 @@
-                    MoveSingulo(singularity, physics);
-                }
+                MoveSingulo(singularity, physics);
             }
         }
 
         private void MoveSingulo(ServerSingularityComponent singularity, PhysicsComponent physics)
         {
-            // To prevent getting stuck, ServerSingularityComponent will zero the velocity of a singularity when it goes to a level <= 1 (see here).
-            if (singularity.Level <= 1) return;
-            // TODO: Could try gradual changes instead but for now just try to replicate
+            // TODO: Need to make this events instead.
+            if (singularity.Level <= 1)
+            {
+                physics.BodyStatus = BodyStatus.OnGround;
+                return;
+            }
 
+            // TODO: Could try gradual changes instead
             var pushVector = new Vector2(_robustRandom.Next(-10, 10), _robustRandom.Next(-10, 10));
 
             if (pushVector == Vector2.Zero) return;
 
-            physics.LinearVelocity = Vector2.Zero;
-            physics.LinearVelocity = pushVector.Normalized * 2;
-        }
-
-        private void PullEntities(ServerSingularityComponent component)
-        {
-            var singularityCoords = component.Owner.Transform.Coordinates;
-            // TODO: Maybe if we have named fixtures needs to pull out the outer circle collider (inner will be for deleting).
-            var entitiesToPull = IoCManager.Resolve<IEntityLookup>().GetEntitiesInRange(singularityCoords, component.Level * 10);
-            foreach (var entity in entitiesToPull)
-            {
-                if (!entity.TryGetComponent<PhysicsComponent>(out var collidableComponent) || collidableComponent.BodyType == BodyType.Static) continue;
-                if (entity.HasComponent<GhostComponent>()) continue;
-                if (singularityCoords.EntityId != entity.Transform.Coordinates.EntityId) continue;
-                var vec = (singularityCoords - entity.Transform.Coordinates).Position;
-                if (vec == Vector2.Zero) continue;
-
-                var speed = 10 / vec.Length * component.Level;
-
-                collidableComponent.ApplyLinearImpulse(vec.Normalized * speed);
-            }
-        }
-
-        private void DestroyTiles(ServerSingularityComponent component)
-        {
-            if (!component.Owner.TryGetComponent(out PhysicsComponent? physicsComponent)) return;
-            var worldBox = physicsComponent.GetWorldAABB();
-
-            foreach (var grid in _mapManager.FindGridsIntersecting(component.Owner.Transform.MapID, worldBox))
-            {
-                foreach (var tile in grid.GetTilesIntersecting(worldBox))
-                {
-                    grid.SetTile(tile.GridIndices, Tile.Empty);
-                }
-            }
+            physics.BodyStatus = BodyStatus.InAir;
+            physics.ApplyLinearImpulse(pushVector.Normalized + (1f / singularity.Level) * physics.Mass);
```

Comment:
No resetting inertia after each tick? That would be more true to SS13 singulos but I don't know if you want that.